src/lightgbm/src/main/python/LightGBMClassifier.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in project root for information.
+
+import sys
+from pyspark import SQLContext
+from pyspark import SparkContext
+
+if sys.version >= '3':
+    basestring = str
+
+from mmlspark._LightGBMClassifier import _LightGBMClassifier
+from mmlspark._LightGBMClassifier import M
+from pyspark.ml.common import inherit_doc
+
+@inherit_doc
+class LightGBMClassifier(_LightGBMClassifier):
+    def _create_model(self, java_model):
+        model = LightGBMClassificationModel()
+        model._java_obj = java_model
+        model._transfer_params_from_java()
+        return model
+
+@inherit_doc
+class LightGBMClassificationModel(M):
+    def save_booster(self, sparkSession, filename):

```

Comment:
could you name this saveNativeModel? I know it grinds the pythonic gears, but its the API spark ML follows.